config/locales/ar.yml

```diff
@@ -33,7 +33,7 @@ ar:
   groups:
     group_header: اعثر على أقرب مجموعة إليك
     learn_more: اعرف أكثر عن هذه المجموعة
-    next_event: "Next event: %{event_name} (in %{time_in_future})"
+    next_event: "الفعاليات القادمة: %{event_name} (in %{time_in_future})"
```

Comment:
The word 'in' within brackets needs to be translated in this string too; this template gets rendered as "Next event: Intro meeting (in 3 days)". You might also need to rearrange where the brackets are in this sentence too. `time_in_future` will automatically handle rendering strings like "3 days" in Arabic. 

```
next_event: "الفعاليات القادمة: %{event_name} (in %{time_in_future})"
                                               ↑↑
```